test/test_helper.rb

```diff
@@ -1,5 +1,5 @@
 require "simplecov"
-SimpleCov.start
+SimpleCov.start 'rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.